Marlin/Configuration.h

```diff
@@ -2382,7 +2382,7 @@
 // If you have a speaker that can produce tones, enable it here.
 // By default Marlin assumes you have a buzzer with a fixed frequency.
 //
-#define SPEAKER
+// #define SPEAKER
```

Comment:
I am not sure about this change, without it I was getting an error:
```
Marlin/src/HAL/AVR/../../inc/../HAL/AVR/inc/SanityCheck.h:63:4: error: #error "FAN_PIN 9 Hardware PWM uses Timer 2 which conflicts with Arduino AVR Tone Timer (for SPEAKER)."
Marlin/src/HAL/AVR/../../inc/../HAL/AVR/inc/SanityCheck.h:64:4: error: #error "Disable SPEAKER or enable FAN_SOFT_PWM."
```

Have you tried compiling it in the newest version?